src/core/game/game_processor/game_processor.standard.ts

```diff
@@ -1870,12 +1921,12 @@ export class StandardGameProcessor extends GameProcessor {
           }
         } else if (toArea === CardMoveArea.OutsideArea) {
           to.getCardIds(
-            (toArea as unknown) as PlayerCardsArea,
+            toArea as unknown as PlayerCardsArea,
             Precondition.exists(toOutsideArea, 'outside area must have an area name'),
           ).push(...actualCardIds);
         } else if (toArea === CardMoveArea.HandArea) {
           this.room.transformCard(to, actualCardIds, PlayerCardsArea.HandArea);
-          to.getCardIds((toArea as unknown) as PlayerCardsArea).push(...actualCardIds);
+          to.getCardIds(toArea as unknown as PlayerCardsArea).push(...actualCardIds);
```

Comment:
can you create a map between `PlayerCardsArea` and `GameCardArea`? It's not a proper way to use `as unknown as` all the time to force cast the variable types.

Reply:
I have not modified here, this is formatting,
and line 1927 has clarified the scope of toArea
see line 1927 `if (toArea === CardMoveArea.HandArea)`